amp/pom.xml

```diff
@@ -1600,6 +1600,27 @@
                 </executions>
             </plugin>
 
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>1.1.1</version>
+                <executions>
+                    <execution>
+                        <phase>compile</phase>
```

Comment:
Please name this run. It is much easier to understand build output that way.
Ex: `<id>Generate API Errors</id>`